### PR TITLE
updated piecewise linear T schedule

### PR DESCRIPTION
**Pull request by M-R-Schaefer**
changed the `steps` parameter to `duration`, which is the number of time steps between temperatures.

### CODE REVIEW

**Comment by M-R-Schaefer** [comment]
closes #353

File: apax/md/schedules.py [hunk 1]
```diff
@@ -17,13 +17,16 @@ def __call__(self, step) -> float:
...
+        self.temperatures = jnp.array(temperatures)
+        steps = np.cumsum(durations)
         self.steps = jnp.array(steps)
 
     def __call__(self, step) -> float:
-        T = jnp.interp(step, self.steps, self.values, left=self.T0, right=self.values[-1])
+        T = jnp.interp(
+            step, self.steps, self.temperatures, left=self.T0, right=self.temperatures[-1]
+        )
         return T * units.kB
```
Comment (PythonFZ) [inline apax/md/schedules.py:30]: You made the changes to MD config also in https://github.com/apax-hub/apax/pull/352 but the changes in this file are missing. Can this cause issues if https://github.com/apax-hub/apax/pull/352 is merged first? Tests should cover this?

Comment (M-R-Schaefer) [reply apax/md/schedules.py:30]: Yes the test does actually fail rn but should be fixed with the other PR

Comment (M-R-Schaefer) [reply apax/md/schedules.py:30]: Not sure what I was thinking here

**Pull request merged**